jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -39,6 +39,9 @@
 import com.ibm.oti.reflect.TypeAnnotationParser;
 
 /*[IF Sidecar19-SE]*/
+/*[IF JAVA_SPEC_VERSION >= 20]*/
+import java.io.InputStream;
+/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
```

Comment:
FYI: @JasonFengJ9

Reply:
Moved `initialSystemIn` into `System`, also assigned its value within `completeInitialization()` instead of `setIn()` which is a public method and could be invoked by an application.

Reply:
Picked up Jason's changes at https://github.com/eclipse-openj9/openj9/pull/16574.